datafusion/core/src/physical_plan/file_format/parquet.rs

```diff
@@ -1635,27 +1740,38 @@ mod tests {
 
     #[tokio::test]
     async fn parquet_page_index_exec_metrics() {
-        let c1: ArrayRef = Arc::new(Int32Array::from(vec![Some(1), None, Some(2)]));
-        let c2: ArrayRef = Arc::new(Int32Array::from(vec![Some(3), Some(4), Some(5)]));
+        let c1: ArrayRef = Arc::new(Int32Array::from(vec![
```

Comment:
This was the only test that used the "merge multiple batches together" behavior of `store_parquet` -- so I rewrote the tests to inline the creation and ensure we got evenly created two row pages